### PR TITLE
Ketchup ganttchart implemention

### PR DESCRIPTION
**Pull request by dev01prishusoft**

### CODE REVIEW

File: packages/ketchup/src/components/kup-planner/utils/kup-grid-renderer/kup-grid-renderer.tsx [hunk 1]
```diff
@@ -0,0 +1,920 @@
...
+import { addToDate } from "../kup-planner-renderer-helper";
+import { handleTaskBySVGMouseEvent, getProgressPoint } from '../helpers/bar.helpers';
+import { isKeyboardEvent } from '../helpers/other.helpers';
+import { hexToCSSFilter } from 'hex-to-css-filter';
+@Component({
+    tag: 'kup-grid-renderer',
+    styleUrl: 'kup-grid-renderer.scss',
+    shadow: false, // Enable Shadow DOM
+})
+export class KupGirdRenderer {
```
Comment (lucafoscili) [inline packages/ketchup/src/components/kup-planner/utils/kup-grid-renderer/kup-grid-renderer.tsx:12]: Typo - Gird should be Grid

File: packages/ketchup/src/components/kup-planner/utils/kup-planner-renderer.data.ts [hunk 2]
```diff
@@ -0,0 +1,1526 @@
...
+    //         mainGanttPlanner1PropsMock.items = mainGanttPlaner1ItemsAfterClick;
+    //     } else {
+    //         mainGanttPlanner1PropsMock.items = mainGanttPlaner1Items;
+    //     }
+    //     isClicked = !isClicked;
+    // }
+};
+// // planner 1
+
+const mainGanttPlaner1Items: any = [
```
Comment (lucafoscili) [inline packages/ketchup/src/components/kup-planner/utils/kup-planner-renderer.data.ts:36]: Typo - Planer should be Planner

File: packages/ketchup/src/components/kup-planner/utils/kup-planner-renderer.data.ts
Comment (lucafoscili) [inline packages/ketchup/src/components/kup-planner/utils/kup-planner-renderer.data.ts:1]: This file is used for mock ups, it shouldn't be inside the component but included in the testing page (planner.html + assets/planner.js)

**Review by lucafoscili — CHANGES_REQUESTED** [review]
In addition to the comments, please remove commented lines of code (I don't mean real comments, but old lines removed with //).

File: .yarn/install-state.gz
[the remark this answers is not in the document]
Comment (dev01prishusoft) [reply .yarn/install-state.gz:1]: Fix and Pushed

**Review by lucafoscili — APPROVED** [review]

**Pull request merged**